dev/com.ibm.ws.channel.ssl/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -35,3 +35,5 @@ default.sslRef.desc=The default SSL configuration repertoire. The default value
 ssl.sslSessionTimeout=SSL session timeout
 ssl.sslSessionTimeout.desc=The timeout limit for an SSL session that is established by the SSL Channel. 
 
+ssl.sslSessionCacheSize=SSL session Cache Size
```

Comment:
Suggested change: `SSL session cache size` (Use lowercase unless uppercase is needed.)

Reply:
Updated to lowercase as suggested